konf/site.yaml

```diff
@@ -29,9 +29,30 @@ extraHosts:
 # Overrides for production
 production:
   replicas: 5
+  routes:
+    - paths: [/blog]
+      name: canonical-com-blog
+      app_name: canonical.com-blog
+      image: prod-comms.docker-registry.canonical.com/canonical.com
+      replicas: 3
+      memoryLimit: 512Mi
```

Comment:
Can we start at `256Mi` and see how it goes?

Reply:
Thanks, updated.